### PR TITLE
v9.8 changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/deploy/mattermost-changelog.md [hunk 1]
```diff
@@ -11,6 +11,109 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+#### Administration
+ - Downloading a support packet is now extensible with plugins. If a plugin can add content to the support packet, it will be displayed in the commercial support modal. Administrators will have the option to include/exclude that from the support package.
+ - Added Channel Bookmarks permissions to the channel user role and to the channel moderation system.
+ - Added progress logs for attachments in bulk exports.
+ - Added a **System Console** option to rebuild Elasticsearch channels indexes.
+ - Obfuscated ``ReplicaLagSettings`` in the Support Packet.
+ - Improved license loading errors.
+ - Updated the keycloak docker configs and added a ``make`` command.
+ - Removed unused ``IsOAuth`` field from ``AppError``.
+ - ``bool`` is now used for ``license_is_tria``l in the Support Packet.
```
Comment (Movion) [inline source/deploy/mattermost-changelog.md:52]: Little typo here - i think it should be ``license_is_trial``

**Review by Movion — COMMENTED** [review]
I just found one typo while scrolling trough the release notes.

File: source/deploy/mattermost-changelog.md (hunk 1 above)
Comment (amyblais) [reply source/deploy/mattermost-changelog.md]: Thank you!

File: source/deploy/mattermost-changelog.md [hunk 2]
```diff
@@ -11,6 +11,110 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+
+ #### Changes to Professional and Enterprise plans:
+ - Under ``ExperimentalSettings`` in ``config.json``:
+   - Removed the ``UseNewSAMLLibrary`` experimental setting.
+
+### Go Version
+ - v9.8 is built with Go ``v1.21.0``.
+
+### Known Issues
+ - Status may sometimes get stuck as **Away** or **Offline** in High Availability mode with IP Hash turned off.
```
Comment (nab-77) [inline source/deploy/mattermost-changelog.md:108]: If this refers to this https://mattermost.atlassian.net/browse/MM-57153 then the HA part is not relevant.

**Review by nab-77 — CHANGES_REQUESTED** [review]
Comment on clarity of a known issue

**Review by cpoile — APPROVED** [review]

File: source/deploy/mattermost-changelog.md (hunk 2 above)
Comment (amyblais) [reply source/deploy/mattermost-changelog.md]: Thank you, I've edited the note.

File: source/deploy/mattermost-changelog.md [hunk 3]
```diff
@@ -11,6 +11,110 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Added emoji tooltips on hover in post message.
+ - Made the appearance of several tooltips more consistent.
+ - Updated theme colors for onboarding tour points.
+ - Updated the right-hand side Thread view to use relative timestamps to be more consistent with the global Threads view.
+ - Added a total reply count to the right-hand side thread view.
+ - Added Channel Bookmarks (disabled by default).
+
+#### Administration
+ - Downloading a support packet is now extensible with plugins. If a plugin can add content to the support packet, it will be displayed in the commercial support modal. Administrators will have the option to include/exclude that from the support package.
+ - Added a warning in the announcement banner for system admins when the system is over the post limit.
```
Comment (marianunez) [inline source/deploy/mattermost-changelog.md:45]:
```suggestion
 Added safety limit error message in compiled Team Edition and Enterprise Edition deployments when enterprise scale and access control automation features are unavailable and message count exceeds 5 million posts. ERROR_SAFE_LIMITS_EXCEEDED.
```

**Review by marianunez — APPROVED** [review]

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:

File: source/deploy/mattermost-changelog.md [hunk 4]
```diff
@@ -11,6 +11,110 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+   - Increased the default payload size limit (``MaximumPayloadSizeBytes``) from 100 kB to 300 kB.
+ - Under ``ClusterSettings`` in ``config.json``:
+   - Removed unused settings ``StreamingPort``, ``MaxIdleConns``, ``MaxIdleConnsPerHost`` and ``IdleConnTimeoutMilliseconds``.
+
+ #### Changes to Professional and Enterprise plans:
+ - Under ``ExperimentalSettings`` in ``config.json``:
+   - Removed the ``UseNewSAMLLibrary`` experimental setting.
+
+### Go Version
+ - v9.8 is built with Go ``v1.21.0``.
```
Comment (agnivade) [inline source/deploy/mattermost-changelog.md:105]: 1.21.8. Can you clarify from where you found that it's 1.21.0? We should probably get it fixed.

Comment (amyblais) [reply source/deploy/mattermost-changelog.md:105]: https://github.com/mattermost/mattermost/pull/26512 - dot release version wasn't mentioned in the release note.

Comment (agnivade) [reply source/deploy/mattermost-changelog.md]: Oh okay. I thought we agreed to check the `server/build/Dockerfile.buildenv` file for the exact version. But anyways, my bad on the release notes.

**Review by agnivade — APPROVED** [review]

File: source/deploy/mattermost-changelog.md (hunk 4 above)
Comment (amyblais) [reply source/deploy/mattermost-changelog.md]: I forgot to, thank you 👍

File: source/deploy/mattermost-changelog.md [hunk 5]
```diff
@@ -11,6 +11,110 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
...
+ - Updated minimum required Edge and Chrome versions to 122+.
+
+```{Important}
+If you upgrade from a release earlier than v9.7, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Upgraded Nodejs to v20.11.
```
Comment (jespino) [inline source/deploy/mattermost-changelog.md:28]: This is not UI related.

Comment (amyblais) [reply source/deploy/mattermost-changelog.md:28]: I moved it to the Administration section.

**Review by gabrieljackson — APPROVED** [review]

**Review by streamer45 — APPROVED** [review]

**Review by saturninoabril — APPROVED** [review]
@amyblais Thanks!

Let's get this merge soon since the https://docs.mattermost.com/upgrade/extended-support-release.html#what-is-the-life-cycle-of-an-extended-support-release is a bit outdated already.

**Comment by amyblais** [comment]
> @amyblais Thanks!
> 
> Let's get this merge soon since the https://docs.mattermost.com/upgrade/extended-support-release.html#what-is-the-life-cycle-of-an-extended-support-release is a bit outdated already.

@saturninoabril This PR is planned to be merged on May 16th which is when v9.8 is released and v8.1 ESR goes out of support.

**Comment by saturninoabril** [comment]
Thanks @amyblais, totally understand since this is for 9.8. I wonder if in the future, the update for `source/images/ESR_graphic2.png` can be done monthly so the image is regularly up-to-date.

**Pull request merged**